worlds/rogue_legacy/Regions.py

```diff
@@ -1,15 +1,18 @@
-from typing import Dict, List, NamedTuple, Optional
+from typing import Dict, List, NamedTuple, Optional, TYPE_CHECKING
 
 from BaseClasses import MultiWorld, Region, Entrance
 from .Locations import RLLocation, location_table, get_locations_by_category
 
+if TYPE_CHECKING:
+    from . import RLWorld
+
 
 class RLRegionData(NamedTuple):
     locations: Optional[List[str]]
     region_exits: Optional[List[str]]
 
 
-def create_regions(multiworld: MultiWorld, player: int):
+def create_regions(world: "RLWorld", player: int):
```

Comment:
Won't block the PR over this (could be considered out of scope), but in general, if we're passing world, I'd rather `player` just be grabbed from `world.player`, than passed as another arg